autoapi/mappers/dotnet.py

```diff
@@ -180,16 +180,16 @@ def output_rst(self, root, source_suffix):
                 filename = obj.name.split('(')[0]
             except IndexError:
                 filename = id
-            filename = filename.replace('#', '-')
+            filename = filename.replace('#', '-').replace('<', '-').replace('>', '')
```

Comment:
Agreed. A simpler approach would be a regex with non-match replacements to catch everything.